olx_parser.py

```diff
@@ -1,124 +1,181 @@
+from typing import TypedDict, NotRequired
+import logging
+
 import requests
 from bs4 import BeautifulSoup
 from datetime import datetime, timedelta
 
+logger = logging.getLogger()
+
+month_mapping = {'январь': 'January',
+                 'февраль': 'February',
+                 'март': 'March',
+                 'апрель': 'April',
+                 'май': 'May',
+                 'июнь': 'June',
+                 'июль': 'July',
+                 'август': 'August',
+                 'сентябрь': 'September',
+                 'октябрь': 'October',
+                 'ноябрь': 'November',
+                 'декабрь': 'December',
+                 'января': 'January',
+                 'февраля': 'February',
+                 'марта': 'March',
+                 'апреля': 'April',
+                 'мая': 'May',
+                 'июня': 'June',
+                 'июля': 'July',
+                 'августа': 'August',
+                 'сентября': 'September',
+                 'октября': 'October',
+                 'ноября': 'November',
+                 'декабря': 'December'}
+
+class Adv(TypedDict):
+    url: str
+    id: str 
+    ads_name: str
+    ads_content: str
+    phone_number: NotRequired[str]
+    seller_name: str
+    date_registered: datetime
+    date_of_last_visit: datetime
+    date_posted: datetime
+    number_of_looks: NotRequired[int]
+    location: NotRequired[str]
+    picture: str
+
+def translate_month_to_en(ru_month_name: str) -> str:
+    # Функция перевода месяца с RU на EN
+    return month_mapping[ru_month_name]
+
+
+def parser_page(url):
```

Comment:
функции принято называть глаголами
```suggestion
def parse_category(url):
```

Reply:
done